sdjournal/read.go

```diff
@@ -44,19 +44,27 @@ type JournalReaderConfig struct {
 	// If not empty, the journal instance will point to a journal residing
 	// in this directory. The supplied path may be relative or absolute.
 	Path string
+
+	// If not nil, Formatter will be used to translate the resulting entries
+	// into strings. If not set, the default format (timestamp and message field)
+	// will be used.
+	Formatter func(entry *JournalEntry) string
```

Comment:
I would prefer to make this more error-friendly by returning a `(string, error)` instead. As a consumer, would it be ok?